<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>404</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>551</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>文件下载</string>
   </property>
   <widget class="QProgressBar" name="pgbFileDownload">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>420</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFileDownload">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始下载</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>35</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>正在下载，请稍等......</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>551</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>文件拷贝</string>
   </property>
   <widget class="QProgressBar" name="pgbFileCopy">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>420</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFileCopy">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始拷贝</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>42</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>正在拷贝，请稍等......</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>551</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>网络请求</string>
   </property>
   <widget class="QProgressBar" name="pgbNetwork">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>42</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>网络请求中，请稍等......</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
